src/Pages/Employer/EmployerProfilePage.jsx

```diff
@@ -1,15 +1,52 @@
+/* eslint-disable */
 import React from 'react';
 import {
   LoggedInHeader,
   EditProfileCard,
   ReviewCard,
-  SkillsRequiredCard,
+  SkillsRequiredCards,
   Footer,
 } from '../../components';
 import {hrProfile} from '../../components/Assets';
 import {Link} from 'react-router-dom';
 
 const EmployerProfilePage = () => {
+  const handleEditJobsClick = () => {
+    
+    const jobId = localStorage.getItem('job_id');
+    const accessToken = localStorage.getItem('access_token');
+
+    const requestData = {
```

Comment:
This data should come from the form. Also we should use Formik for Forms.
https://formik.org/docs/overview